core/src/main/resources/jenkins/model/Jenkins/EnforceSlaveAgentPortAdministrativeMonitor/message.properties

```diff
@@ -1 +1,2 @@
-description=Slave Agent Port has been manually changed. Its value will be reset to {0} on restart.
+description=Slave Agent Port has been manually changed. Its value will be reset to {0,number,#} on restart.
+reset=Reset to {0,number,#}
```

Comment:
*… now*; to emphasize what it does as opposed to what happens automatically as mentioned in the description preceding this button.